jina/peapods/__init__.py

```diff
@@ -1,8 +1,7 @@
 __copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from .peas import BasePea
+from .peas import BasePea as Pea
```

Comment:
This disables importing of `BasePea` from `jina.peapods`. Is this intentional? I think it makes sense, just wanted to double check.

Reply:
I have no strong opinion on this. If you want I can change it.